main.js

```diff
@@ -0,0 +1,140 @@
+let player = {
+  player1: {
+    name: "player1",
+    letter: "X",
+    winVal: 1 //when the abs(sum) of a row/col or diagonal is 3 the win condition is triggered
+  },
+  player2: {
+    name: "player2",
+    letter: "O",
+    winVal: -1
+  },
+  curentPlayer: "player1",
+  nextPlayer: "player2"
+};
+
+let start = document.getElementById("start1Player");
+let AIstart = document.getElementById("AIstart");
+let status = document.getElementById("turnIdentifier");
+let cells = document.getElementsByClassName("cell");
+let timer = document.getElementById("timer");
+let time = 0;
+
+AIstart.addEventListener("click", () => {
+  player.AI = true;
+  time = 0;
```

Comment:
You should probably set `time = 0` in your `startGame` function like you do with the rest of your initial variables to keep all your initial values together.